src/patternfly/components/DataList/data-list.scss

```diff
@@ -384,8 +384,14 @@
 
 // Expandable content
 .pf-c-data-list__expandable-content {
+  max-height: var(--pf-c-data-list__expandable-content--MaxHeight);
+  overflow-y: auto;
   box-shadow: var(--pf-c-data-list__expandable-content--BoxShadow);
 
+  @media screen and (min-width: $pf-global--breakpoint--md) {
+    max-height: initial;
```

Comment:
Can you set `overflow: visible;` here? Otherwise it will hide overflow by default, resulting in something like this (an expanded dropdown in expandable content) 

<img width="455" alt="Screen Shot 2020-03-17 at 6 19 34 PM" src="https://user-images.githubusercontent.com/35148959/76910445-0063df00-687c-11ea-9621-70a9f0093a47.png">

Reply:
good catch, thanks for finding that